assets/dev/js/editor/document/hooks/data/document/elements/delete/section-columns.js

```diff
@@ -1,63 +0,0 @@
-import After from 'elementor-api/modules/hooks/data/after';
-
-export class SectionsColumns extends After {
-	getCommand() {
-		return 'document/elements/delete';
-	}
-
-	getId() {
-		return 'delete-section-columns';
-	}
-
-	getContainerType() {
-		return 'column';
-	}
-
-	getConditions( args ) {
-		const { containers = [ args.container ] } = args;
-
-		// On `document/elements/move` do not fire the hook!.
-		return ! $e.commands.isCurrentFirstTrace( 'document/elements/move' ) && containers.some( ( container ) =>
```

Comment:
Please check:
1. Move a column in the same section from left to right and vise versa.
2. Move a column from a section to another.
3. Move a column from a section to an inner-section and vice versa.